dask_cloudprovider/providers/azure/tests/test_azureml_gpu.py

```diff
@@ -0,0 +1,168 @@
+from azureml.core import Workspace, Experiment, Datastore, Dataset, Environment
+from azureml.core.runconfig import MpiConfiguration
+from azureml.core.compute import ComputeTarget, AmlCompute
+from azureml.core.authentication import InteractiveLoginAuthentication
+from dask_cloudprovider import AzureMLCluster
+from ..azureml_configs import AzureMLConfigurations
+import os
+
+import asyncio
+class aml_test:
+    def __init__(self):
+        # interactive_auth = InteractiveLoginAuthentication(
+        #     tenant_id='72f988bf-86f1-41af-91ab-2d7cd011db47'
+        # )
+
+        subscription_id = '6560575d-fa06-4e7d-95fb-f962e74efd7a'

```

Comment:
we should make an "official" azure ml workspace in its own RG for setting up an automated test - we should schedule the test on our end to pull from here and run @FredFLi -

Reply:
and we shouldn't hard code these things, should use a config.json - then doesn't matter which ws the test is running in - or better yet, do it from an azure ml run -> get everything from run context -> spin up resources, test -> clean up

Reply:
Agree. will do it.